integration-tests/tests/test_waku_rpc.py

```diff
@@ -0,0 +1,109 @@
+import random
+import pytest
+import jsonschema
+import json
+import time
+from conftest import option, user_1, user_2
+from test_cases import RpcTestCase, TransactionTestCase
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@pytest.mark.rpc
+@pytest.mark.wakuext
+class TestRpc(RpcTestCase):
+
+    @pytest.mark.parametrize(
+        "method, params",
+        [
+            ("wakuext_peers", []),
+        ],
+    )
+    def test_(self, method, params):
+        _id = str(random.randint(1, 8888))
+
+        response = self.rpc_request(method, params, _id, url=option.rpc_url_2)
+        self.verify_is_valid_json_rpc_response(response)
+        with open(f"{option.base_dir}/schemas/{method}", "r") as schema:
+            jsonschema.validate(instance=response.json(), schema=json.load(schema))
```

Comment:
I'd rather change that to be a 1-line method, something like
```suggestion
            self.validate_json_schema(instance=response, schema=method)
```